templates/mod/ssl.conf.erb

```diff
@@ -1,8 +1,16 @@
 <IfModule mod_ssl.c>
   SSLRandomSeed startup builtin
-  SSLRandomSeed startup file:/dev/urandom 512
   SSLRandomSeed connect builtin
+<% if @ssl_random_seeds and ! @ssl_random_seeds.empty? -%>
```

Comment:
in general, we set the pre-existing default options as *default* value for the class parameter.
which you appear to have already done, since it's about *additional* values of SSLRandomSeed